content/en/docs/concepts/services-networking/service.md

```diff
@@ -117,8 +117,10 @@ spec:
       targetPort: 9376
 ```
 
-Applying this manifest creates a new Service named "my-service", which
-targets TCP port 9376 on any Pod with the `app.kubernetes.io/name: MyApp` label.
+Applying this manifest creates a new Service named "my-service" with default 
+[service type](/docs/concepts/services-networking/service/#publishing-services-service-types)
```

Comment:
👋 @T-Lakshmi .
What do you think about using the in-page link (#publishing-services-service-types) and creating two sentences instead of one?

Applying this manifest creates a new Service named "my-service" with the default Cluster IP 
[service type](#publishing-services-service-types). The Service targets TCP port 9376 on any Pod with the `app.kubernetes.io/name: MyApp` label.

Reply:
Hi @kbhawkey,
Alrigh!
I missed to use in-page link here, let me update as per your suggestions.